docs: adjust guidance on build content bundle to mention Registry Connect (#6493)
spectrocloud/librarium: docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md

## Limitation

:::warning

If your cluster uses the legacy **Harbor Edge-Native Config** pack, you cannot use the Palette CLI to build content
bundles or export cluster definitions. You must use the Palette Edge CLI.

Refer to
[Migrate From Harbor Edge-Native Config Pack](../../site-deployment/deploy-custom-registries/migrate-edge-native-config.md)

> to learn how to migrate off the legacy pack to **Registry Connect** pack. This pack does not require a local Harbor

Reviewer: ```suggestion
to learn how to migrate from the legacy pack to the **Registry Connect** pack. The **Registry Connect** pack does not require a local Harbor
```
For clarity :)

registry.

:::

<!-- prettier-ignore-start -->
- You can configure built-in registry packs or your own custom OCI registry. Both configurations require the **Registry

>   Connect** pack. Built-in registries must be configured using either <VersionedLink text="Harbor" url="/integrations/packs/?pack=harbor" /> pack or the <VersionedLink text="Zot" url="/integrations/packs/?pack=zot" /> pack. Custom registries allow you to configure [custom packs](../../../../registries-and-packs/add-custom-packs.md). For more information, refer to [Deploy Cluster with External Registry](../../site-deployment/deploy-custom-registries/deploy-external-registry.md)

Reviewer: ```suggestion
  Connect** pack. Built-in registries must be configured using either the <VersionedLink text="Harbor" url="/integrations/packs/?pack=harbor" /> pack or the <VersionedLink text="Zot" url="/integrations/packs/?pack=zot" /> pack. Custom registries allow you to configure [custom packs](../../../../registries-and-packs/add-custom-packs.md). For more information, refer to [Deploy Cluster with External Registry](../../site-deployment/deploy-custom-registries/deploy-external-registry.md)
```
